CONTRIBUTING.md

```diff
@@ -0,0 +1,88 @@
+# Contributing instructions
+
+Thank you for considering to contribute to the Covid19 Dashboard Vaccini repository!
+Here you will find instructions and guidelines for contributing efficiently.
+
+Before proceeding further, please review our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Issues
+
+If you find an issue please check if it's already known by searching
+the [issue section](https://github.com/italia/covid19-dashboard-vaccini/issues), otherwise
+file a new one: we really appreciate it! :rocket:.
+
+You're welcome to contribute to open issues with more information or by adding
+:+1: on them: it will help the maintainers identify the issues to be prioritized.
+
+### Creating a new issue
+
+There are three templates for new issues: `Bug report`, `Feature request`
+and `General issue`, pick the relevant one and follow the instructions
+in that template.
```

Comment:
We don't have templates on this repo

Reply:
Let's get rid of that part then, we'll insert those templates later on, thanks